field_setting/field_setting_parser.py

```diff
@@ -297,27 +284,37 @@ def str_to_bool(bool_str):
 	"""
 	if bool_str.lower() in ("0", "f", "false", "n", "no"):
 		return False
+
 	if bool_str.lower() in ("1", "t", "true", "y", "yes"):
 		return True
+
 	raise ValueError("\"" + str(bool_str)
 		+ "\" does not match a boolean value.")
 
 
 if __name__ == "__main__":
-	try:
-		field_setting_path = Path(argv[1])
-	except IndexError:
-		print("The path to a field setting file must "
-			+ "be provided as an argument.")
-		exit()
-
-	try:
-		print_val_type = str_to_bool(argv[2])
-	except IndexError:
-		print_val_type = False
-	except ValueError as ve:
-		print(ve)
-		exit()
+	from argparse import ArgumentParser
+
+	parser = ArgumentParser(description=
+		"This module allows to extract from a YAML file the values to write\
+		in an ÉTS club expense report. The YAML file must comply with the\
+		format of those provided in this repository. If this module is\
+		executed and given the path to a YAML file as an argument, it will\
+		print the name of the report's fields in the console with the value\
+		they are assigned.")
```

Comment:
```suggestion
	parser = ArgumentParser(description=__doc__)
```

Tu peux directement référencer le docstring du module courant :)